pallets/ddc-verification/src/tests.rs

```diff
@@ -7,9 +7,20 @@ use sp_core::{
 	},
 	Pair, H256,
 };
-use sp_io::TestExternalities;
+use sp_io::{hashing::blake2_128, TestExternalities};
+use sp_runtime::AccountId32;
 
-use crate::{mock::*, Error, Event, NodeActivity, *};
+use crate::{mock::*, ConsensusError, Error, Event, NodeActivity, *};
+
+fn get_id_for_node_actity(activity: &NodeActivity) -> [u8; 16] {
+	blake2_128(&activity.node_id)
+}
+
+fn get_id_for_customer_activity(activity: &CustomerActivity) -> [u8; 16] {
+	let mut data = activity.customer_id.to_vec();
+	data.extend_from_slice(&activity.bucket_id.encode());
+	blake2_128(&data)
```

Comment:
```suggestion
	blake2_128(&activity.customer_id)
```

Why we are encoding and extending from slice here? Customer id and node id both have same type.  Can we use `blake2_128(&activity.customer_id)`?

Reply:
we need hash of (bucket_id + cystoimer_id)